ironic/inspector.ipxe

```diff
@@ -0,0 +1,9 @@
+#!ipxe
+
+
+:retry_boot
+echo In inspector.ipxe
+imgfree
+kernel --timeout 60000 http://172.22.0.1/images/ironic-python-agent.kernel ipa-inspection-callback-url=http://172.22.0.1:5050/v1/continue ipa-inspection-collectors=default,extra-hardware,numa-topology,logs systemd.journald.forward_to_console=yes BOOTIF=${mac} ipa-debug=1 ipa-inspection-dhcp-all-interfaces=1 ipa-collect-lldp=1 initrd=ironic-python-agent.initramfs || goto retry_boot
```

Comment:
Let's remove `extra-hardware,numa-topology` at least for now. It can add a minute or two to introspection. Or do we actually need the extra hardware information from `python-hardware`?

Reply:
We can probably remove it initially - is there some docs reference where we can see exactly what data is collected by default, and what is added with extra-hardware?

I'm thinking in due course we'll want stuff like lldp data to validate networking, but initially probably make/model/cpu/ram/disks will be enough to match a basic profile of the expected platform.

Reply:
Okay, just keep in mind that LLDP collection also makes inspection noticeably longer.

Reply:
Removed extra-hardware,numa-topology